app/src/main/java/com/sun/heartrate/ui/heartbeat/HeartbeatFragment.kt

```diff
@@ -81,18 +91,63 @@ class HeartbeatFragment : Fragment(), HeartbeatContract.View, View.OnClickListen
         }
     }
     
-    private fun upDateIuButton(isCheckCamera: Boolean) {
-        if (isCheckCamera) {
-            buttonStart.text = getText(R.string.label_pause)
-        } else {
-            buttonStart.text = getText(R.string.label_start)
+    private fun updateUiScreenHeartbeat() {
+        heartbeatPresenter.run {
+            showButtonLabel(isRecording())
+            displayImageViewHeart(isRecording())
+            updateProgressBar(isRecording())
+        }
+        displayHeatRate(0)
+        displayGuideline(R.string.title_measuring_heart_rate)
+        
+    }
+    
+    private fun showButtonLabel(isRecording: Boolean) {
+        buttonStart?.text = if (isRecording) getText(R.string.label_pause)
+        else getText(R.string.label_start)
+    }
+    
+    private fun updateProgressBar(isRecording: Boolean) {
+        countDownProgressBar.cancel()
+        progressBarTime?.progress = 0
+        if (isRecording) {
```

Comment:
Với các biểu thức điều kiện chỉ thực hiện một lệnh thì Kotlin suggest viết cùng dòng thế này nhé.
```suggestion
        if (isRecording) countDownProgressBar.start()
```